src/commands/synthetics/__tests__/reporters/junit.test.ts

```diff
@@ -253,14 +322,202 @@ describe('Junit reporter', () => {
           },
         },
       }
-      const suite = reporter['getTestCase'](resultMock, MOCK_BASE_URL)
-      expect(suite.$).toMatchObject({
-        ...getDefaultStats(),
-        errors: 2,
-        failures: 1,
-        tests: 3,
-        warnings: 1,
+      const testCase = reporter['getTestCase'](resultMock, '')
+      expect(testCase.$).toMatchObject({
+        ...getDefaultTestCaseStats(),
+        steps_count: 3,
+        steps_errors: 2,
+        steps_failures: 1,
+        steps_warnings: 1,
       })
     })
   })
 })
+
+describe('GitLab test report compatibility', () => {
+  const writeMock: Writable['write'] = jest.fn()
+  const commandMock: Args = {
+    context: ({stdout: {write: writeMock}} as unknown) as BaseContext,
+    jUnitReport: 'junit',
+    runName: 'Test Suite name',
+  }
+
+  let reporter: JUnitReporter
+
+  beforeEach(() => {
+    reporter = new JUnitReporter(commandMock as RunTestCommand)
+  })
+
+  it('all test case names are unique', () => {
```

Comment:
Nit: we could use `test` instead of `it` (`it` is generally used for "it should ..." sentences, though it doesn't really matter in the end)

Reply:
I know it's a _nit_ but you reacted with a thumb-up so I guess you forgot to update these `it` to `test`? 😛

Reply:
Oh I got lost and made the commit in the [subsequent PR](https://github.com/DataDog/datadog-ci/pull/648) 😅